tsconfig.json

```diff
@@ -1,24 +1,22 @@
 {
     "compilerOptions": {
         "target": "es5",
-        "module": "umd",
+        "module": "es2015",
         "moduleResolution": "node",
         "emitDecoratorMetadata": true,
         "experimentalDecorators": true,
+        "removeComments": true,
         "noImplicitAny": false,
-        "outDir": "build",
         "declaration": true,
         "sourceMap": true,
         "allowSyntheticDefaultImports": true,
         "lib": [
             "es2016",
             "dom"
-        ]
-    },
-    "include": [
-        "**/*.ts"
-    ],
-    "exclude": [
-        "node_modules"
-    ]
+        ],
+        "baseUrl": "packages",
+        "paths": {
+            "@morgan-stanley/desktopjs": ["desktopjs/src"]
```

Comment:
Without this I kept hitting an issue where references to the desktopjs package could not be resolved in vscode